code/.eslintcache

```diff
@@ -0,0 +1 @@
edDeprecatedRules":"43"},{"filePath":"44","messages":"45","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"28"},{"filePath":"46","messages":"47","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"28"},{"filePath":"48","messages":"49","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"28"},{"filePath":"50","messages":"51","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"28"},"/Users/bunnyhop/Documents/1_RecordKeeping/WRK/d.School/_Ref.MakeWeb/Technigo/Sprint-4/Week-16/project-labyrinth-1/code/src/index.js",[],["52","53","54","55","56","57"],"/Users/bunnyhop/Documents/1_RecordKeeping/WRK/d.School/_Ref.MakeWeb/Technigo/Sprint-4/Week-16/project-labyrinth-1/code/src/App.js",[],"/Users/bunnyhop/Documents/1_RecordKeeping/WRK/d.School/_Ref.MakeWeb/Technigo/Sprint-4/Week-16/project-labyrinth-1/code/src/reducers/game.js",[],"/Users/bunnyhop/Documents/1_RecordKeeping/WRK/d.School/_Ref.MakeWeb/Technigo/Sprint-4/Week-16/project-labyrinth-1/code/src/reducers/ui.js",[],"/Users/bunnyhop/Documents/1_RecordKeeping/WRK/d.School/_Ref.MakeWeb/Technigo/Sprint-4/Week-16/project-labyrinth-1/code/src/lib/Button.js",[],"/Users/bunnyhop/Documents/1_RecordKeeping/WRK/d.School/_Ref.MakeWeb/Technigo/Sprint-4/Week-16/project-labyrinth-1/code/src/components/Loading.js",[],"/Users/bunnyhop/Documents/1_RecordKeeping/WRK/d.School/_Ref.MakeWeb/Technigo/Sprint-4/Week-16/project-labyrinth-1/code/src/components/Labyrinth.js",[],"/Users/bunnyhop/Documents/1_RecordKeeping/WRK/d.School/_Ref.MakeWeb/Technigo/Sprint-4/Week-16/project-labyrinth-1/code/src/lib/Card.js",[],["58","59","60","61","62","63"],"/Users/bunnyhop/Documents/1_RecordKeeping/WRK/d.School/_Ref.MakeWeb/Technigo/Sprint-4/Week-16/project-labyrinth-1/code/src/lib/Input.js",[],"/Users/bunnyhop/Documents/1_RecordKeeping/WRK/d.School/_Ref.MakeWeb/Technigo/Sprint-4/Week-16/project-labyrinth-1/code/src/lib/Container.js",[],"/Users/bunnyhop/Documents/1_RecordKeeping/WRK/d.School/_Ref.MakeWeb/Technigo/Sprint-4/Week-16/project-labyrinth-1/code/src/components/Start.js",[],"/Users/bunnyhop/Documents/1_RecordKeeping/WRK/d.School/_Ref.MakeWeb/Technigo/Sprint-4/Week-16/project-labyrinth-1/code/src/reducers/fetch.js",[],{"ruleId":"64","replacedBy":"65"},{"ruleId":"66","replacedBy":"67"},{"ruleId":"68","replacedBy":"69"},{"ruleId":"70","replacedBy":"71"},{"ruleId":"72","replacedBy":"73"},{"ruleId":"74","replacedBy":"75"},{"ruleId":"64","replacedBy":"76"},{"ruleId":"66","replacedBy":"77"},{"ruleId":"68","replacedBy":"78"},{"ruleId":"70","replacedBy":"79"},{"ruleId":"72","replacedBy":"80"},{"ruleId":"74","replacedBy":"81"},"lines-around-directive",["82"],"no-spaced-func",["83"],"global-require",[],"no-buffer-constructor",[],"no-new-require",[],"no-path-concat",[],["82"],["83"],[],[],[],[],"padding-line-between-statements","func-call-spacing"]
```

Comment:
This file should not be included in the repository

Reply:
removed.